x/shield/types/types.go

```diff
@@ -79,8 +79,8 @@ type Provider struct {
 	TotalLocked sdk.Coins `json:"total_locked" yaml:"total_locked"`
 	// amount of coins staked but not in any pool
 	Available sdk.Int `json:"available" yaml:"available"`
-	// amount of collateral that is in withdrawable queue
-	Withdrawal sdk.Int `json:"withrawal" yaml:"withdrawal"`
+	// amount of collateral that is in withdraw queue
+	Withdraw sdk.Int `json:"withrawal" yaml:"withdrawal"`
```

Comment:
Since we call it `Withdrawing` in `Collateral`, should we also name it `Withdrawing` here?

Reply:
Agreed. Ill make the change